scripts/clean-up-localizations.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const glob = require('glob');
+
+const localizationFile_en = 'src/assets/i18n/en_US.json';
+const localizationFile_fr = 'src/assets/i18n/fr_FR.json';
+const localizationFile_de = 'src/assets/i18n/de_DE.json';
```

Comment:
I'd rather not hard-code this, but make it generic. I guess we can expect `en_US.json` to always be there, but other files should be discovered automatically.